pkg/http/api.go

```diff
@@ -485,6 +503,15 @@ func (a *api) onGetSecret(reqCtx *fasthttp.RequestCtx) {
 	metadata := getMetadataFromRequest(reqCtx)
 
 	key := reqCtx.UserValue(secretNameParam).(string)
+
+	if !a.isSecretAllowed(secretStoreName, key) {
+		msg := NewErrorResponse(
+			"ERR_PERMISSION_DENIED",
+			fmt.Sprintf("Access denied by policy to get %s from %s", key, secretStoreName))
+		respondWithError(reqCtx, 403, msg)
```

Comment:
more readable const for 403

General comment for the entire PR

Reply:
Creating a separate issue for this would be better than making changes for the http/api.go fully in this PR.